<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="hu-hu">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">9bfaded869bf5ae7415300830c81eb7c33395cf0</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.Data.Design.TypedDataSetGeneratorException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">78e81a26ddaef89d478990b1e557e40ff8defa54</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">22752cf8c6133107650f37b1d987f619af996f62</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown when a name conflict occurs while a strongly typed <bpt id="p1">&lt;xref href="System.Data.DataSet"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> is being generated.</source>
          <target state="translated">A kivétel fordul elő, amikor neve ütközés lép fel egy szigorú típusmegadású <bpt id="p1">&lt;xref href="System.Data.DataSet"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> létrehozása folyamatban van.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source>The TypedDataSetGeneratorException class indicates that a conflict occurred while an attempt is being made to generate a typed dataset class.</source>
          <target state="translated">A TypedDataSetGeneratorException osztály azt jelzi, hogy ütközés történt kísérlet történik a típusos dataset osztály létrehozásához.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a system-supplied message that describes the error.</source>
          <target state="translated">Egy új példányát inicializálja a <bpt id="p1">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> osztály rendszer által megadott üzenetet, amely leírja a hibát.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor initializes the &lt;xref:System.Exception.Message%2A&gt; property of the new instance to a system-supplied message that describes the error, such as "DefaultMessageDisplayedByParameterlessCtorWriterMustSupply" This message takes into account the current system culture.</source>
          <target state="translated">Ez a konstruktor inicializálja a &lt;xref:System.Exception.Message%2A&gt;tulajdonság egy rendszer által megadott üzenetet, amely leírja a hibát, például a "DefaultMessageDisplayedByParameterlessCtorWriterMustSupply" ezt az üzenetet az új példány veszi figyelembe a rendszer aktuális kulturális környezetének.&lt;/xref:System.Exception.Message%2A&gt;</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.InsufficientMemoryException&gt;.</source>
          <target state="translated">Az alábbi táblázat a kezdeti tulajdonságértékek &lt;xref:System.InsufficientMemoryException&gt;.&lt;/xref:System.InsufficientMemoryException&gt; példánya</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|       The &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt; class indicates that a conflict occurred while an attempt is being made to generate a typed dataset class.</source>
          <target state="translated">| Tulajdonság |} Érték |}   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| <ph id="ph1">`null`</ph>.|   | &lt;xref:System.Exception.Message%2A&gt;| Az üzenet honosított hibakarakterlánc. |}       Az &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;osztály azt jelzi, hogy ütközés történt, miközben egy típusos dataset osztály létrehozására tett kísérlet alatt&lt;/xref:System.Data.Design.TypedDataSetGeneratorException&gt; &lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class by passing in a collection of errors.</source>
          <target state="translated">Egy új példányát inicializálja a <bpt id="p1">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> történő hibák gyűjtemény osztályhoz.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt; class indicates that a conflict occurred while an attempt is being made to generate a typed dataset class.</source>
          <target state="translated">Az &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;osztály azt jelzi, hogy ütközés történt kísérlet egy típusos dataset osztály létrehozása történik&lt;/xref:System.Data.Design.TypedDataSetGeneratorException&gt;</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve">
          <source>An &lt;xref:System.Collections.IList&gt; of errors.</source>
          <target state="translated">Egy &lt;xref:System.Collections.IList&gt;hibák.&lt;/xref:System.Collections.IList&gt;</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified message that describes the error.</source>
          <target state="translated">Egy új példányát inicializálja a <bpt id="p1">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> osztály egy adott üzenet, amely leírja a hibát.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;.</source>
          <target state="translated">Az alábbi táblázat a kezdeti tulajdonságértékek &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;.&lt;/xref:System.Data.Design.TypedDataSetGeneratorException&gt; példánya</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string specified in <ph id="ph2">`message`</ph>.|       The &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt; class indicates that a conflict occurred while an attempt is being made to generate a typed dataset class.</source>
          <target state="translated">| Tulajdonság |} Érték |}   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| <ph id="ph1">`null`</ph>.|   | &lt;xref:System.Exception.Message%2A&gt;| A hiba üzenet mezőben megadott karakterlánc <ph id="ph2">`message`</ph>. |}       Az &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;osztály azt jelzi, hogy ütközés történt, miközben egy típusos dataset osztály létrehozására tett kísérlet alatt&lt;/xref:System.Data.Design.TypedDataSetGeneratorException&gt; &lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>The message that describes the exception.</source>
          <target state="translated">A kivétel leíró üzenet.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class, using the specified serialization information and streaming context.</source>
          <target state="translated">Egy új példányát inicializálja a <bpt id="p1">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> osztály, használja a megadott szerializálási adat szerepel, és adatfolyam-környezetben.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">Ez a konstruktor nevezik meglévő kölcsönökre az adatfolyam továbbított kivétel objektum deszerializálása közben.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/add/includes/ajax-current-ext-md.md)</ept>.The &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt; class indicates that a conflict occurred while an attempt is being made to generate a typed dataset class.</source>
          <target state="translated">További információkért lásd: <bpt id="p1">[</bpt>XML és a SOAP-szerializálási<ept id="p1">](~/add/includes/ajax-current-ext-md.md)</ept>. Az &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;osztály azt jelzi, hogy ütközés történt kísérlet egy típusos dataset osztály létrehozása történik&lt;/xref:System.Data.Design.TypedDataSetGeneratorException&gt;</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve">
          <source>A <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> object.</source>
          <target state="translated">A <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> object.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve">
          <source>A &lt;xref:System.Runtime.Serialization.StreamingContext&gt; structure.</source>
          <target state="translated">A &lt;xref:System.Runtime.Serialization.StreamingContext&gt;struktúra.&lt;/xref:System.Runtime.Serialization.StreamingContext&gt;</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with the specified string and inner exception.</source>
          <target state="translated">Egy új példányát inicializálja a <bpt id="p1">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> osztály a megadott karakterlánc és a belső kivételben találhatók.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">Kivételt vált ki, mert egy előző kivétel közvetlen eredménye tartalmaznia kell egy hivatkozást az előző kivétel az a &lt;xref:System.Exception.InnerException%2A&gt;tulajdonság.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or <ph id="ph1">`null`</ph> if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">A &lt;xref:System.Exception.InnerException%2A&gt;tulajdonság a konstruktornak átadott ugyanazt az értéket adja vissza vagy <ph id="ph1">`null`</ph> Ha a &lt;xref:System.Exception.InnerException%2A&gt;tulajdonság nem adja meg a belső kivétel értékét a konstruktor.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;.</source>
          <target state="translated">Az alábbi táblázat a kezdeti tulajdonságértékek &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;.&lt;/xref:System.Data.Design.TypedDataSetGeneratorException&gt; példánya</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string specified in <ph id="ph2">`message`</ph>.|       The &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt; class indicates that a conflict occurred while an attempt is being made to generate a typed dataset class.</source>
          <target state="translated">| Tulajdonság |} Érték |}   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| <ph id="ph1">`null`</ph>.|   | &lt;xref:System.Exception.Message%2A&gt;| A hiba üzenet mezőben megadott karakterlánc <ph id="ph2">`message`</ph>. |}       Az &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;osztály azt jelzi, hogy ütközés történt, miközben egy típusos dataset osztály létrehozására tett kísérlet alatt&lt;/xref:System.Data.Design.TypedDataSetGeneratorException&gt; &lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve">
          <source>The message that describes the exception.</source>
          <target state="translated">A kivétel leíró üzenet.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">A kivétellel, hogy a jelenlegi kivétel okozza.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve">
          <source>If the <ph id="ph1">`innerException`</ph> parameter is not <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>, the current exception is raised in a <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> block that handles the inner exception.</source>
          <target state="translated">Ha a <ph id="ph1">`innerException`</ph> paraméter nem <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>, a jelenlegi kivétel a következik be egy <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt> <ept id="p2">&lt;/xref&gt;</ept> blokk, amely kezeli a belső kivételben találhatók.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve">
          <source>Gets a dynamic list of generated errors.</source>
          <target state="translated">Generált hibák dinamikus listájának lekérése.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt; class indicates that a conflict occurred while an attempt is being made to generate a typed dataset class.</source>
          <target state="translated">Az &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;osztály azt jelzi, hogy ütközés történt kísérlet egy típusos dataset osztály létrehozása történik&lt;/xref:System.Data.Design.TypedDataSetGeneratorException&gt;</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve">
          <source>The error list.</source>
          <target state="translated">A hibalistát.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve">
          <source>Implements the <bpt id="p1">&lt;xref uid="langword_csharp_ISerializable" name="ISerializable" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> interface and returns the data that you must have to serialize the <bpt id="p2">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> object.</source>
          <target state="translated">Megvalósítja a <bpt id="p1">&lt;xref uid="langword_csharp_ISerializable" name="ISerializable" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> felületet, és visszaadja az adatokat, amelyek szerializálni kell rendelkeznie a <bpt id="p2">&lt;xref href="System.Data.Design.TypedDataSetGeneratorException"&gt;</bpt> <ept id="p2">&lt;/xref&gt;</ept> objektum.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">Ez a konstruktor nevezik meglévő kölcsönökre az adatfolyam továbbított kivétel objektum deszerializálása közben.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/add/includes/ajax-current-ext-md.md)</ept>.</source>
          <target state="translated">További információkért lásd: <bpt id="p1">[</bpt>XML és a SOAP-szerializálási<ept id="p1">](~/add/includes/ajax-current-ext-md.md)</ept>.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt; class indicates that a conflict occurred while an attempt is being made to generate a typed dataset class.</source>
          <target state="translated">Az &lt;xref:System.Data.Design.TypedDataSetGeneratorException&gt;osztály azt jelzi, hogy ütközés történt kísérlet egy típusos dataset osztály létrehozása történik&lt;/xref:System.Data.Design.TypedDataSetGeneratorException&gt;</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve">
          <source>A <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> object.</source>
          <target state="translated">A <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> object.</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve">
          <source>A &lt;xref:System.Runtime.Serialization.StreamingContext&gt; structure.</source>
          <target state="translated">A &lt;xref:System.Runtime.Serialization.StreamingContext&gt;struktúra.&lt;/xref:System.Runtime.Serialization.StreamingContext&gt;</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>